### DISCUSSION TITLE
Add Azure Authentication to MSSQL Datasource

### DISCUSSION THREAD

**Discussion by baldm0mma** [grafana/grafana#69085] [Data Sources]
In late April 2023, the Azure SCOM MI team approached us to see if we could build out some enhancements to our MSSQL plugin. Currently, the plugin works well with Azure SQL Database, and any on-prem SQL services they have, but cannot connect with Azure SQL MIs. Essentially our team would need to build the Azure Authentication piece that will allow Azure SQL MI users to connect safely to their data.

Currently, the offering the Azure SCOM MI team is rolling out is not supported by our MSSQL plugin, and therefore users cannot access any of their MSSQL data; there is an Azure Authentication piece missing.

**Comment by baldm0mma** [comment]
Relevant documentation:
- [Current specs](https://docs.google.com/document/d/1ST9Fzl033cKCnPg3NTv7JWTCW8kx82TtNW2xbs7eEOM/edit)
- [Design Doc](https://docs.google.com/document/d/1stn7rqY6_OJy0lIA4kOnQY2CFAGbpTh1iTo4uHu6EYE/edit#heading=h.5sybau7waq2q)

**Comment by gelicia** [comment]
There was a closed issue for this discussion, so I believe it is resolved. Closing.
